elements/pl-integer-input/pl-integer-input.py

```diff
@@ -34,46 +44,47 @@ def prepare(element_html, data):
         "base",
         "allow-blank",
         "blank-value",
+        "placeholder",
+        "show-score",
     ]
 
     pl.check_attribs(element, required_attribs, optional_attribs)
     name = pl.get_string_attrib(element, "answers-name")
     base = pl.get_integer_attrib(element, "base", BASE_DEFAULT)
 
     if base != 0 and (base < 2 or base > 36):
-        raise Exception("Base must be either 0, or between 2 and 36")
+        raise ValueError(f"Base must be either 0, or between 2 and 36, not {base}")
```

Comment:
I know this didn't change here, but what exactly is base 0? 👀

Reply:
https://prairielearn.readthedocs.io/en/latest/elements/#specifying-a-non-trivial-base